ScheduleLesson/Program.cs

```diff
@@ -38,6 +38,7 @@ public static void Main(string[] args)
 
 
             app.MapControllers();
+            app.UseMiddleware<ExceptionHandlingMiddleware>();
```

Comment:
This is not the ideal place to add this ExceptionHandlingMiddleware. 
.NET 8 pipeline process